packages/iov-encoding/src/encoding.spec.ts

```diff
@@ -313,3 +313,21 @@ describe("Encoding", () => {
     });
   });
 });
+
+describe("Bech32", () => {
+  const lskAddressUtf8 = Encoding.toUtf8("6472030874529564639L");
+  const riseAddressUtf8 = Encoding.toUtf8("6472030874529564639R");
+  const ethAddressUtf8 = Encoding.toUtf8("0x9D4e856E572E442f0A4b2763e72d08A0E99D8deD");
```

Comment:
I think this example is very irritating because you don't encode the 20 bytes Ethereum address but the 42 bytes string. Better go for `const ethAddressRaw = Encoding.fromHex("9D4e856E572E442f0A4b2763e72d08A0E99D8deD")`